.circleci/config.yml

```diff
@@ -62,23 +56,66 @@ jobs:
           name: mgmt - Validate Terraform
           command: cd terraform/mgmt && terraform validate
 
-      - run:
-          name: env - Create Terraform variables file
-          command: cd terraform/env && cp terraform.tfvars.example terraform.tfvars
       - run:
           name: env - Set up Terraform
-          command: cd terraform/env && terraform init -backend=false
+          command: cd terraform/env && terraform init "-backend-config=bucket=$TF_ENV_BUCKET"
       - run:
           name: env - Validate Terraform
           command: cd terraform/env && terraform validate
+      - persist_to_workspace:
+          root: .
+          paths:
+            - terraform/env/.terraform/*
+
+  deploy_env:
+    docker:
+      - image: 18fgsa/devsecops-builder:alpine
+    steps:
+      - attach_workspace:
+          at: .
+
+      - add_ssh_keys
+      - run:
+          name: Disable host key checking
+          command: printf "\nHost *\n\tStrictHostKeyChecking no" >> ~/.ssh/config
```

Comment:
Possible this isn't needed, or that there's a better way around the problem. I propose we revisit that separately.